### DISCUSSION TITLE
My Github pages stopped to build

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21613] [Other Feature Feedback, Questions, & Ideas]
<p>(Sorry for my grammer)<br>
I have a github pages to use for hosting open source HTML5 games I’ve forked on Github, and I use it to host some flash games too.<br>
Today my Github pages failed to build with only this warning:<br>
"Unable to build page, please try again later. "<br>
The page can successfully builded before, but after I adding something there, it won’t build anymore.<br>
I don’t want to add a .nojekyll file because I use both Jekyll and Static HTML to build this page, so I hope someone would help it out. Thanks!<br>
BTW here’s my repo: <a href="https://github.com/sfmemz/sfmemz.github.io" class="inline-onebox" rel="noopener nofollow ugc">GitHub - sfmemz/sfmemz.github.io</a></p>

**Comment by isaxk** [comment]
<p>This is the same problem I have:</p><aside class="quote quote-modified" data-post="1" data-topic="180157">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/github-pages-is-not-working-on-any-of-my-repos/180157">Github Pages is not working on ANY of my repos</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    So, I use GitHub pages for my Jekyll website. But I also like the feature where you can have multiple repos on the same url. Example: 
repo-name: exampeuser.github.io, url: exampeuser.github.io. 
repo-name: project, url: exampeuser.github.io/project/ 
So I go to edit one of my non-main repos and all I get is 
Your site is having problems building: Unable to build page. Please try again later.  
So I try to edit one of my other repos and the it’s the same responce. 
I cleared my browser cache and…
  </blockquote>
</aside>

**Comment by DGCK81LNN** [comment]
<p>I’m having the same problem and I’ve already created a <code>.nojekyll</code> long ago, so that doesn’t seem to be the problem.</p>

**Comment by wmxzrs** [comment]
<p>i have same problem！</p>

**Comment by ghost** [comment, marked as answer]
<p>Nvm my github page was builded successfully today, this might be a server error</p>

**Comment by wmxzrs** [comment]
<p>yeah，but i deleted my repo that is  deployed by gitpages because i thought it maybe my repo’ mistake of setting. :sweat:</p>

**Comment by ghost** [comment]
<p>Then recreate it if you have a backup</p>
